sherpa/models/model.py

```diff
@@ -735,14 +737,14 @@ def _set_thawed_pars_maxes(self, vals):
             v = SherpaFloat(v)
             if v < p.hard_min:
                 p.max = p.hard_min
-                warning(('value of parameter %s maximum is below ' +
-                         'hard minimum; ' +
-                         'setting to hard minimum') % p.fullname)
+                warning('value of parameter %s maximum is below '
+                        'hard minimum; setting to hard minimum',
+                        p.fullname)
```

Comment:
Could go to an f-string here, too.

Reply:
We could. However, pylint complains about this as this is a logger, and it knows that you can "save time" by not interpolating the string for those cases where the logger is going to drop the message. AKA https://vald-phoenix.github.io/pylint-errors/plerr/errors/logging/W1201.html

Although I admit it's not a useful time saving here, I was just trying to avoid too many red squiggles from my lint checks!